observation_portal/urls.py

```diff
@@ -60,6 +60,7 @@
 api_urlpatterns = ([
     re_path(r'^', include(router.urls)),
     re_path(r'^api-token-auth/', obtain_auth_token, name='api-token-auth'),
+    path('users-bulk/', BulkCreateUsersApiView.as_view(), name='users-bulk'),
```

Comment:
Can we not use /api/users/ since its internal? I see nothing exists there now for us.

Reply:
What do you mean by it's internal?